src/store/internals/removeEnabledElement.js

```diff
@@ -98,6 +100,17 @@ const _removeEnabledElement = function(enabledElement) {
   }
 };
 
+/**
+ * Remove load handler for the enabled element
+ * @private
+ * @method
+ * @param {HTMLElement} enabledElement
```

Comment:
JSDoc is wrong, this should be `element`

Reply:
done